lib/grammers-session/build.rs

```diff
@@ -37,8 +46,20 @@ fn main() -> std::io::Result<()> {
         ..Default::default()
     };
 
-    generate_rust_code(&mut file, &definitions, CURRENT_VERSION, &config)?;
-    file.flush()?;
+    generate_rust_code(
+        &mut common_file,
+        &mut types_file,
+        &mut functions_file,
+        &mut enums_file,
+        &definitions,
+        CURRENT_VERSION,
+        &config,
+    )?;
```

Comment:
Way too many parameters of the same type now (4 files in a row). Perhaps we should construct a struct to make sure we don't accidentally write to the wrong file during a refactor.